src/patternfly/components/Wizard/wizard-nav-link-icon.hbs

```diff
@@ -0,0 +1,6 @@
+<span class="{{pfv}}wizard__nav-link-icon{{#if wizard-nav-link-icon--modifier}} {{wizard-nav-link-icon--modifier}}{{/if}}"
```

Comment:
I might rename this from icon to status-icon, just so we can leave room for a step icon that we can target uniquely from the status icon and not run into a namespace issue. WDYT?